README.md

```diff
@@ -6,7 +6,9 @@ Confluent's .NET Client for Apache Kafka<sup>TM</sup>
 [![Chat on Slack](https://img.shields.io/badge/chat-on%20slack-7A5979.svg)](https://confluentcommunity.slack.com/messages/clients)
 
 **confluent-kafka-dotnet** is Confluent's .NET client for [Apache Kafka](http://kafka.apache.org/) and the
-[Confluent Platform](https://www.confluent.io/product/).
+[Confluent Platform](https://www.confluent.io/product/), which is a more complete distribution of
+Apache Kafka available in two editions - [Open Source](https://www.confluent.io/product/confluent-open-source/) and
```

Comment:
I don't think this is quite the wording we want. e.g. this is what's currently on our home page
> The free, open-source streaming platform based on Apache Kafka , Confluent Platform is the best way to get started with real-time data streams.